### PR TITLE
feat(actions): Added gdrive action type as alias to webhook

### PR DESCRIPTION
**Pull request by Geo25rey**
I need to set GDRIVE_PDF_UPLOADER_URL to the correct endpoint, but otherwise it works great with relevant tests.

### CODE REVIEW

**Comment by Geo25rey** [comment]
Check failed due to FIXME comment.

File: packages/main/package-lock.json [hunk 1]
```diff
@@ -46,37 +46,6 @@
         "@babel/highlight": "^7.12.13"
       }
     },
-    "@babel/core": {
```
Comment (AquiGorka) [inline packages/main/package-lock.json:49]: Is this correct? I don't see a reason for these changes to modify the deps lock.

File: packages/main/src/commands/actions/add.ts [hunk 2]
```diff
@@ -9,6 +9,8 @@ import { askVerifyEmailFlow } from '../../utils/askVerifyEmailFlow'
 import { askVerifySmsFlow } from '../../utils/askVerifySmsFlow'
 import withStandardErrors from '../../utils/errorHandling'
 
+const GDRIVE_PDF_UPLOADER_URL = 'gdrive.url' // FIXME what's the url??
```
Comment (AquiGorka) [inline packages/main/src/commands/actions/add.ts:12]: `https://us-central1-mailscript-firebase.cloudfunctions.net/` + path/to/function

**Review by AquiGorka — APPROVED** [review]
Looks good to me, I do want @kanej to go over it since he's the one most familiar with the `cli` context.

File: packages/main/package-lock.json (hunk 1 above)
Comment (Geo25rey) [reply packages/main/package-lock.json:49]: Not sure. I didn't manually change it.

File: packages/main/src/commands/actions/add.ts (hunk 2 above)
Comment (Geo25rey) [reply packages/main/src/commands/actions/add.ts]: Done

File: packages/main/package-lock.json (hunk 1 above)
Comment (AquiGorka) [reply packages/main/package-lock.json:49]: Can you remove this file from the PR then?

**Comment by Geo25rey** [comment]
@kanej Not sure why lint test is failing after running `yarn format`

**Review by kanej — APPROVED** [review]
Get the lint error fixed and it looks good. We aren't going to merge until we have the `google app` authorised though.

**Pull request merged**